tests/integration/targets/github_key/aliases

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+azp/posix/1
```

Comment:
I don't think doing invalid requests got GitHub API and to random domains we don't own on the net is OK, so please mark this test as `unsupported` so it won't run in CI.
```suggestion
unsupported
```